### PR TITLE
Feat: create runnable Django project

### LINKED ISSUE #5: Create runnable Django project
Depends on #4 

- [ ] Add latest Django 5.x as dependency in `pyproject.toml`
- [ ] Initialize new Django project rooted in the `pems/` top-level directory
- [ ] Read sensitive settings from env, add basic `.env.sample`
- [ ] Use SQLite database settings for now
- [ ] `launch.json` to allow runing the site on built-in server using F5

### PR DESCRIPTION
**Pull request by lalver1**
Closes #5
This PR sets up a minimal runnable Django project. It includes:

- adding Django as a dependency
- initializing the PeMS Django project using a SQLite database
- setting up VS Code for Python projects (including setting up the debugger)

### CODE REVIEW

File: pems/settings.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+# Static files (CSS, JavaScript, Images)
+# https://docs.djangoproject.com/en/5.1/howto/static-files/
+
+STATIC_URL = "static/"
+
+# Default primary key field type
+# https://docs.djangoproject.com/en/5.1/ref/settings/#default-auto-field
+
+DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
```
Comment (thekaveman) [inline pems/settings.py:118]: Can we make this a [`UUIDField`](https://docs.djangoproject.com/en/5.1/ref/models/fields/#uuidfield) instead?

File: .env.sample [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# Django storage
+DJANGO_STORAGE_DIR=.
+DJANGO_DB_FILE=django.db
```
Comment (thekaveman) [inline .env.sample:3]: Let's add `*.db` to the `.gitignore` and `.dockerignore`

File: .vscode/settings.json [hunk 3]
```diff
@@ -11,5 +11,9 @@
   "files.trimTrailingWhitespace": true,
   "[markdown]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode"
-  }
+  },
+  "[python]": {
+    "editor.defaultFormatter": "ms-python.black-formatter"
+  },
+  "python.languageServer": "Pylance"
```
Comment (thekaveman) [inline .vscode/settings.json:18]: Can we add

```json
"[django-html][html]": {
    "editor.defaultFormatter": "monosans.djlint",
    "djlint.enableLinting": true,
    "djlint.profile": "django"
},
"workbench.editorAssociations": {
    "*.db": "sqlite-viewer.option"
}
```

File: .devcontainer/devcontainer.json [hunk 4]
```diff
@@ -32,6 +32,9 @@
         "eamodio.gitlens",
         "esbenp.prettier-vscode",
         "mhutchie.git-graph",
+        "ms-python.python",
+        "ms-python.black-formatter",
+        "ms-python.flake8",
```
Comment (thekaveman) [inline .devcontainer/devcontainer.json:37]: Can we add 

```
"batisteo.vscode-django",
"monosans.djlint",
"qwtel.sqlite-viewer",
```

**Review by thekaveman — CHANGES_REQUESTED** [review]

File: pems/settings.py (hunk 1 above)
Comment (lalver1) [reply pems/settings.py]: It made sense and I changed it to [`UUIDField`](https://docs.djangoproject.com/en/5.1/ref/models/fields/#uuidfield) in a5d86a2905492c1736ad5249dfd49356b90fb74f but I got `ValueError: Primary key 'django.db.models.UUIDField' referred by DEFAULT_AUTO_FIELD must subclass AutoField.` when debugging the site locally.

I noticed that in Benefits we use `UUIDField` for `EnrollmentEvent`, so it's ok for primary keys, but maybe we can't set it to be used as the default?

File: .env.sample (hunk 2 above)
Comment (lalver1) [reply .env.sample:3]: Yep! It's in `.gitignore` and I added it to `.dockerignore` in b049f6a73870384e59c9af538e94e489098806b1

File: .vscode/settings.json (hunk 3 above)
Comment (lalver1) [reply .vscode/settings.json]: For sure, added it in 2a430101f162061360ad05c00ba7c0dde251cb27

File: .devcontainer/devcontainer.json (hunk 4 above)
Comment (lalver1) [reply .devcontainer/devcontainer.json:39]: Yep, added in f20297eeb8aab92500d0209ce55ef0e1e5712e92

File: pems/settings.py (hunk 1 above)
Comment (lalver1) [reply pems/settings.py:118]: Ah, yes that's prob the reason. I removed this change but I agree, we'll use it as our own default.

File: pems/settings.py [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+    }
+}
+
+
+# Password validation
+# https://docs.djangoproject.com/en/5.1/ref/settings/#auth-password-validators
+
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator",
```
Comment (thekaveman) [inline pems/settings.py:84]: Can we add a `.flake8` file, VS Code is complaining about these lines being too long.

See e.g. https://github.com/compilerla/conventional-pre-commit/blob/main/.flake8

**Review by thekaveman — CHANGES_REQUESTED** [review]
Small change, looking good

File: pems/settings.py (hunk 5 above)
Comment (lalver1) [reply pems/settings.py:84]: Thanks! I was trying to figure out where this was set and I completely overlooked it, we now have it in 759fc4ec11fc6b466484e891412f7c1771ef6194

**Review by thekaveman — APPROVED** [review]
Nice!

**Pull request merged**